src/common/exception/src/exception_flight.rs

```diff
@@ -12,17 +12,32 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+use std::sync::Arc;
+
 use common_arrow::arrow_format::flight::data::FlightData;
 
+use crate::exception::ErrorCodeBacktrace;
 use crate::ErrorCode;
 use crate::Result;
 
 impl From<ErrorCode> for FlightData {
     fn from(error: ErrorCode) -> Self {
-        // TODO: has stack trace
+        let error_message = error.message();
```

Comment:
How about encode as `bincode`? Maybe we also need to add some message to indicate that this is a backtrace from other nodes.

Reply:
We also need to be compatible with flight data from older versions of the query.

Reply:
Flight protocol does not need compatibility.

Reply:
> How about encode as `bincode`? Maybe we also need to add some message to indicate that this is a backtrace from other nodes.

This needs to be obtained in the exchange processor. Here we only need to serialize and deserialize errors.